src/main/scala/syntax/GmosNorthFpu.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic.syntax
+
+import gem.enum.GmosNorthFpu
+import gem.enum.GmosNorthFpu._
+
+/**
+ * Syntax extensions for missing properties. These need to be folded back into the Gem enumerations.
+ */
+final class GmosNorthFpuOps(val self: GmosNorthFpu) extends AnyVal {
+
+  def effectiveSlitWidth: Double =
+    self match {
+      case Ifu1          => 0.31
+      case Ifu2          => 0.31
+      case Ifu3          => 0.31
```

Comment:
For the IFUs the value is 0.31 arcsec as given in footnote ª [here](http://www.gemini.edu/sciops/instruments/gmos/spectroscopy-overview/gratings). The others are copied from `slitWidth` which is an `Option` in the model.